alpha/pom.xml

```diff
@@ -35,24 +35,21 @@
     <module>alpha-server</module>
   </modules>
 
+  <properties>
+    <kamon.version>0.6.0</kamon.version>
```

Comment:
We normally put the dependency properties into parent pom, so the other module could share the same properties.